xbmc/video/PlayerController.h

```diff
@@ -54,5 +54,11 @@ class CPlayerController : public ISliderCallback, public IActionListener
   void ShowSlider(int action, int label, float value, float min, float delta, float max, bool modal = false);
 
   int m_sliderAction = 0; ///< \brief set to the action id for a slider being displayed \sa ShowSlider
+
+  /*! \brief Set vertical shift for video image in VideoPlayer
+   \param verticalshift value to set for vertical shift
+   */
```

Comment:
for new code follow new style rules
```suggestion
  /*!
   * \brief Set vertical shift for video image in VideoPlayer
   * \param verticalshift value to set for vertical shift
   */
```